upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -266,6 +266,63 @@ There you can find links to upgrade notes for other versions too.
     - a lot of the possible issues should be already resolved if you followed the previous instruction and ran the `php phing annotations-fix` phing command
     - some of the issues related to class extension need to be addressed manually nevertheless (see the ["Framework extensibility" article](../introduction/framework-extensibility.md#problem-3)) for more information
     - you need to resolve all the other reported problems (it is up to you whether you decide to address them directly or add ignores in your `phpstan.neon`). You can find inspiration in [#1381](https://github.com/shopsys/shopsys/pull/1381) and [#1040](https://github.com/shopsys/shopsys/pull/1040)
+
+- if you want to add stylelint rules to check style coding standards [#1511](https://github.com/shopsys/shopsys/pull/1511)
+    -  add new file [.stylelintignore](https://github.com/shopsys/shopsys/blob/master/project-base/.stylelintignore)
+    -  add new file [.stylelintrc](https://github.com/shopsys/shopsys/blob/master/project-base/.stylelintrc)
+    - update `gruntfile.js.twig` and add new task
+        ```diff
+        + stylelint: {
+        +     all: [
+        +         'src/**/*.less',
+        +     ]
+        + },
+
+          watch: {
+              admin: {
+        ```
+        ```diff
+          {% else -%}
+        -     ['frontendLess{{ domain.id }}']
+        +     ['frontendLess{{ domain.id }}','stylelint']
+          {% endif -%}
+        ```
+        ```diff
+          grunt.loadNpmTasks('grunt-spritesmith');
+        + grunt.loadNpmTasks('grunt-stylelint');
+
+        - grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss", "legacssy"]);
+        + grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss", "legacssy", "stylelint"]);
+        ```
```

Comment:
I don't see legacssy in modified file

Reply:
Why should he be there? I think that we add after legacssy new task - stylelint.

Reply:
Oh, I wasn't quite accurate. 
My point is, in `gruntfile.js.twig` the change is without `legaccsy`
```diff
-   grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss"]);
+   grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss", "stylelint:frontend"]);
```

while in upgrade notes, there is `legacssy` after `postcss`.
```diff
-   grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss", "legacssy"]);
+   grunt.registerTask('default', ["sprite:admin", "sprite:frontend", "webfont", "less", "postcss", "legacssy", "stylelint"]);
```

I think the upgrade notes were written before some merged PR (I suspect #1461)

Reply:
Ou yes, thank you. I understand now and I fixed it.